core/src/config/stump_config.rs

```diff
@@ -65,219 +67,91 @@ use defaults::*;
 ///   let core = StumpCore::new(config).await;
 /// }
 /// ```
-#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+#[derive(StumpConfigGenerator, Serialize, Deserialize, Debug, Clone, PartialEq)]
+#[config_file_location(self.get_config_dir().join("Stump.toml"))]
 pub struct StumpConfig {
 	/// The "release" | "debug" profile with which the application is running.
+	#[default_value("debug".to_string())]
```

Comment:
I think part of the fix for the bug I reference in the 0.0.5 release notes would be to set this to `release` and then the debug default value to `debug`

Reply:
Done, but do note that the `debug()` function is only used in tests - the actual runtime values in debug are set by the `debug_setup()` function in the server's `main.rs` by overriding environment variables.

We may be able to simplify this in the future.